File: Instructor Python/Week02/src/case04/LoopDemoRunner.py
from .LoopDemo import LoopDemo


class LoopDemoRunner():
    def run(self):
        sample = LoopDemo()
        sample.demoForLoop()
        sample.demoWhileLoop()
        sample.demoForEachLoop()
        sample.demoContinueLoop()
        sample.demoBreakLoop()

File: Instructor Java/Week03 Java/src/main/java/case04/LoopDemoRunner.java
package case04;

/**
 * LoopDemoRunner
 */
public class LoopDemoRunner implements Runnable {

  public void run() {
    var sample = new LoopDemo();
    sample.demoForLoop();
    sample.demoWhileLoop();
    sample.demoDoWhileLoop();
    sample.demoForEachLoop();
    sample.demoContinueLoop();
    sample.demoBreakLoop();
  }
}
